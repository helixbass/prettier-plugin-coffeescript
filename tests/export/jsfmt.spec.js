run_spec(__dirname, ['coffeescript'])
run_spec(__dirname, ['coffeescript'], {bracketSpacing: false})
