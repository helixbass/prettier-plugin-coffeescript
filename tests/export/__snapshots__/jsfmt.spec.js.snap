// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`brackets.coffee 1`] = `
export {
  runTaskForChanged, description, someOtherLabel, thatMakes, itGo, multiLine, andMore, soWeCanGetItTo80Columns
}
export {fitsIn, oneLine}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
export {
  runTaskForChanged
  description
  someOtherLabel
  thatMakes
  itGo
  multiLine
  andMore
  soWeCanGetItTo80Columns
}
export { fitsIn, oneLine }

`;

exports[`brackets.coffee 2`] = `
export {
  runTaskForChanged, description, someOtherLabel, thatMakes, itGo, multiLine, andMore, soWeCanGetItTo80Columns
}
export {fitsIn, oneLine}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
export {
  runTaskForChanged
  description
  someOtherLabel
  thatMakes
  itGo
  multiLine
  andMore
  soWeCanGetItTo80Columns
}
export {fitsIn, oneLine}

`;

exports[`empty.coffee 1`] = `
export {}
export {} from '.'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
export {}
export {} from '.'

`;

exports[`empty.coffee 2`] = `
export {}
export {} from '.'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
export {}
export {} from '.'

`;
